Hardware/Hardware/Printer/AirPrintReceipt/ReceiptRenderer.swift

```diff
@@ -242,5 +264,9 @@ private extension ReceiptRenderer {
             "Account Type",
             comment: "Reads as 'Account Type'. Part of the mandatory data in receipts"
         )
+
+        static let totalTaxLineDescription = NSLocalizedString(
+            "Total Tax",
```

Comment:
In the Order Details Payment section we just say Taxes.

![taxes](https://user-images.githubusercontent.com/1595739/135538069-15bf959b-2ac1-4696-8728-21f0bc59854c.jpg)